spyder/plugins/projects/widgets/tests/test_projectdialog.py

```diff
@@ -28,5 +34,42 @@ def test_project_dialog(qtbot):
     assert dlg
 
 
+@pytest.mark.skipif(os.name != 'nt', reason="Specific to Windows plataform")
+def test_projectdialog_location(monkeypatch):
+    """Test that select_location normalizes delimiters and updates the path."""
+    dlg = ProjectDialog(None)
+    mock_getexistingdirectory = Mock()
+    monkeypatch.setattr('spyder.widgets.projects.projectdialog' +
```

Comment:
Hi @vitoriahmc ! I think this is not the correct import. That's why the test is failing in AppVeyor. Instead of `spyder.widgets.projects.projectdialog...` it should be something like `spyder.plugins.projects.widgets.projectdialog...`

Reply:
yeah, sorry, I've just fixed that...